recorder/src/main/cpp/profile_writer.hh

```diff
@@ -86,7 +86,9 @@ private:
 
     recording::Wse cpu_sample_accumulator;
     
-    std::unordered_set<MthId> known_methods;
+    std::unordered_map<MthId, MthId> known_methods;
```

Comment:
Earlier jMethodId returned in call trace was used directly in the method dictionary. Now, since backtraces can be native as well, there is a translation required from jMethodId to an auto-incrementing counter as and when new methods (native or jvm call traces) are discovered.
known_methods keeps a mapping of jMethodId -> translated id.

Maybe known_methods should now be renamed to known_jMethods?